### PR TITLE
updated velero CRDs to velero 1.8.0, fix common.go quay image tags

### PR DESCRIPTION
**Pull request by sseago**

### CODE REVIEW

**Review by dymurray — APPROVED** [review]

File: config/crd/bases/velero.io_volumesnapshotlocations.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
+
+---
```
Comment (dymurray) [inline config/crd/bases/velero.io_volumesnapshotlocations.yaml:2]: Assuming make bundle worked, but just wanted to confirm operator-sdk was happy with this

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**